retrieval_service/datastore/providers/postgres.py

```diff
@@ -425,6 +500,8 @@ async def insert_ticket(
         arrival_airport: str,
         departure_time: str,
         arrival_time: str,
+        seat_row: int | None = None,
```

Comment:
Can we also add test (in postgres_test.py) for insert_tickets and list_tickets since we now have a dataset for it.

Reply:
Great idea, done!